src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -36,7 +36,7 @@ import { Codicon } from 'vs/base/common/codicons';
 import { ActionBar } from 'vs/base/browser/ui/actionbar/actionbar';
 
 export const Context = {
-	Visible: new RawContextKey<boolean>('CodeActionMenuVisible', false, localize('CodeActionMenuVisible', "Whether the code action list widget is visible"))
+	Visible: new RawContextKey<boolean>('codeActionMenuVisible', false, localize('codeActionMenuVisible', "Whether the code action list widget is visible"))
```

Comment:
Fixes a naming convention issue brough up by @jrieken regarding the context key using camel-case.